scene-recognition/requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy==1.26.4
+pandas==2.2.2
+python-dateutil==2.9.0.post0
```

Comment:
looks like `dateutil` and other timezone related dependencies are obsolete now, and can be gone. `six` as well. We don't support py2 anyway.

Reply:
Those are all installed when you pip-install pandas, so whether we like it or not they are going to be there

Reply:
I don't understand. If all others come with pandas, why not just specify pandas as an explicit dependency in the req.txt?

Reply:
There is a difference between (1) just putting in pandas with some and (2) putting in everything via a pip freeze after the panda install. It is partially a matter of taste, I like the latter because it is more precise. We could mention in the readme that a simple pip-install will also suffice.

Reply:
@keighrim Have you worked with requirements.in and pip-compile?

Reply:
No, I haven't.